hummingbot/connector/exchange/kraken/kraken_exchange.pyx

```diff
@@ -822,7 +822,7 @@ cdef class KrakenExchange(ExchangeBase):
         try:
             order_result = None
             order_decimal_amount = f"{decimal_amount:f}"
-            if order_type is OrderType.LIMIT or order_type is OrderType.LIMIT_MAKER:
+            if order_type is OrderType.LIMIT or order_type is OrderType.LIMIT_MAKER or order_type is OrderType.MARKET:
```

Comment:
here you can simplify (mainly if other types are added) with
if order_type in self.supported_order_types() that you added in line 777